content/concepts/did-ddo.md

```diff
@@ -35,97 +70,178 @@ In Ocean, a DID is a string that looks like:
 did:op:0ebed8226ada17fde24b6bf2b95d27f8f05fcce09139ff5cec31f6d81a7cd2ea
 ```
 
+where "0ebed8226ada17fde24b6bf2b95d27f8f05fcce09139ff5cec31f6d81a7cd2ea" = sha256(ERC721 contract addres + chainId)
+
 It follows [the generic DID scheme](https://w3c-ccg.github.io/did-spec/#the-generic-did-scheme).
 
-The part after `did:op:` is the asset's on-chain Ethereum address (minus the "0x"). One can be computed from the other; therefore there is a 1:1 mapping between did and Ethereum address.
 
 ## DDO Attributes
 
-![DDO Content](images/ddo-content.png)
 
-A DDO has these standard attributes:
+A DDO has these standard attributes (required):
+
+- `@context`   = array, contexts used for validation
+- `id`  = string, computed as sha256(address of ERC721 contract + chainId)
+- `created` = contains the date of publishing, ISO Date Time Format yyyy-MM-dd'T'HH:mm:ss. SSSXXX — for example, "2000-10-31T01:30:00.000-05:00
```

Comment:
In the latest DID Spec, the `created` and `updated` properties were moved to inside the `metadata` property.
See for details: https://www.w3.org/TR/did-core/#did-document-metadata

Quoted from https://www.w3.org/TR/did-core/#revision-history
>Changes since the First Public Working Draft include:
>...
>* Separation of DID Document metadata, such as created and updated values, from DID Document properties.

Reply:
not sure about that, in our case `created` and `updated` are filled by `Aquarius` based on tx time (have to confirm this) . If that's the case i would not put them in `metadata` so we can have a clean inheritance between `DDO` and `Asset` (`Aquarius` response)

Reply:
Maybe the structure of `Asset` should change as well, to match the underlying `DDO`?

Also, how closely are we basing our DDO spec off of the w3 DID spec?  Based on @alexcos20 [comment](https://github.com/oceanprotocol/docs/pull/783#discussion_r721260572) I assumed we wanted to "comply" with the w3 DDO spec.

Reply:
getting back to this, as I noticed that we always set the `created` date from the frontend during publish and now wondering what I should do in new publish form.

Moving `created` & `updated` under `metadata` is a simple change so we should do it if we are sure Aquarius is not writing this. To me Aquarius should not fill it as it's against the principles created in beginning of this document. On the frontend we do not send the DDO to Aquarius so why would it fill out things? Frontend has to fill `created` & `updated` on the client side anyway as far as I can see, so should indeed live under `metadata`

Reply:
moved created & updated under metadata.  Aquarius will not touch that.  If you need to timestamp of the event, you can find it in the event object